test/examples/ucp_hello_world.c

```diff
@@ -339,6 +384,11 @@ static int run_ucx_server(ucp_worker_h ucp_worker)
     free(msg);
 
 err_ep:
+    if (err_handling_opt.server_failure) {
+        fprintf(stderr, "Emulation of unexpected failure on server side!!!\n");
+        exit(0);
```

Comment:
You may want to raise(SIGKILL) rather, as exit will call atexit handlers etc, as well as flushing open file descriptors etc, while SIGKILL will more closely resemble a failure.